frontend/packages/prompt-test/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@liam-hq/prompt-test",
+  "private": true,
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "devDependencies": {
+    "@biomejs/biome": "1.9.3",
+    "@langchain/core": "0.3.42",
+    "@liam-hq/configs": "workspace:*",
+    "@liam-hq/jobs": "workspace:*",
+    "dotenv": "16.4.7",
+    "promptfoo": "0.107.7",
+    "tsx": "4.7.1",
+    "typescript": "5"
+  },
+  "scripts": {
+    "build": "tsc",
```

Comment:
Is this necessary?

Reply:
Correct. 🙏  . I think tsc is nessesary , but build is not.

Reply:
https://github.com/liam-hq/liam/pull/1076/commits/88806e9f09c23860d9fe293349ae99d6e606ad9d fixed